<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>749</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Python GUI</string>
  </property>
  <widget class="QLabel" name="lblImage">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>351</width>
     <height>331</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblCannyImage">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>90</y>
     <width>351</width>
     <height>331</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>711</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btnLoadImage">
      <property name="text">
       <string>Load Image</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnSaveImage">
      <property name="text">
       <string>Save Image</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnDetect">
      <property name="text">
       <string>Detect Face + Eyes</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QSlider" name="cannySlider">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>711</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickInterval">
    <number>10</number>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>40</y>
     <width>231</width>
     <height>19</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QCheckBox" name="chbFace">
      <property name="text">
       <string>Face</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="chbEyes">
      <property name="text">
       <string>Eyes</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
